R/pkg/inst/tests/testthat/test_streaming.R

```diff
@@ -61,6 +61,7 @@ test_that("read.stream, write.stream, awaitTermination, stopQuery", {
 
   stopQuery(q)
   expect_true(awaitTermination(q, 1))
+  expect_error(awaitTermination(q), NA)
```

Comment:
`NA` means expects no error ;)
yea, it's not very intuitive. There is no return value in this case, so just checking to make sure it doesn't throw an error